extensions/wikia/Forum/RelatedForumDiscussionController.class.php

```diff
@@ -60,7 +60,7 @@ public static function getData( $articleId ) {
 		$key = wfMemcKey( __CLASS__, 'getData', $articleId );
 		return WikiaDataAccess::cache( $key, self::CACHE_EXPIRY, function() use ( $articleId ) {
 			$wlp = new WallRelatedPages();
-			$messages = $wlp->getArticlesRelatedMessgesSnippet( $articleId, 2, 2 );
+			$messages = $wlp->getArticlesRelatedMessgesSnippet( $articleId, 2 );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$messages". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)